Sources/MessagingPush/RichPush/MessagingPush+RichPush.swift

```diff
@@ -59,7 +59,7 @@ extension MessagingPushImplementation {
 
         logger.info("push was sent from Customer.io. Processing the request...")
 
-        if sdkConfig.autoTrackPushEvents {
+        if moduleConfig.autoTrackPushEvents {
```

Comment:
This `if` statement is OK to be here because we require all customers, including SDK wrapper customers, to initialize the native SDKs in the NSE file before this `if` statement runs. App lifecycle is not a problem for `delivered` metrics.